locator/abstract_replication_strategy.cc

```diff
@@ -234,7 +234,20 @@ abstract_replication_strategy::get_address_ranges(const token_metadata& tm) cons
 future<std::unordered_multimap<inet_address, dht::token_range>>
 abstract_replication_strategy::get_address_ranges(const token_metadata& tm, inet_address endpoint) const {
     std::unordered_multimap<inet_address, dht::token_range> ret;
```

Comment:
Issue references should be added to the cover letter only, no to each patch in the series.

Reply:
An extra reference doesn't hurt (in contrast to a missing one).

Reply:
> Issue references should be added to the cover letter only, no to each patch in the series.

If one cherry picks one of them from the series for backport. It loses the track of the issue immediately.